packages/ui/src/components/vuestic-components/va-dropdown/VaDropdown.vue

```diff
@@ -51,6 +52,7 @@ class DropdownProps {
   closeOnClickOutside = prop<boolean>({ type: Boolean, default: true })
```

Comment:
isContentClosable prop means that you actually have an opportunity to close the dropdown. So if it's false, you can not close the dropdown.

Looks like `close-on-click-inside` makes more sense.

Reply:
> isContentClosable prop means that you actually have an opportunity to close the dropdown. So if it's false, you can not close the dropdown.
> 
> Looks like `close-on-click-inside` makes more sense.
> 
> 🤔

Thanks, yeah - that sounds even clearer and easier to understand.
I'll fix it.

Reply:
As discussed docs have been added, prop is renamed.

If you don't mind I will create a different branch for on slot click behavior because it will probably need docs & description as well. And this issue looks solved already.

Thanks!